designer/server/src/main/scala/pl/touk/nussknacker/ui/api/NotificationApiEndpoints.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.ui.api
+
+import pl.touk.nussknacker.restmodel.BaseEndpointDefinitions
+import pl.touk.nussknacker.restmodel.BaseEndpointDefinitions.SecuredEndpoint
+import pl.touk.nussknacker.security.AuthCredentials
+import pl.touk.nussknacker.ui.notifications.Notification
+import sttp.model.StatusCode.Ok
+import sttp.tapir.json.circe.jsonBody
+import sttp.tapir.{EndpointInput, query, statusCode}
+
+import java.time.Instant
+
+class NotificationApiEndpoints(auth: EndpointInput[AuthCredentials]) extends BaseEndpointDefinitions {
+
+  lazy val notificationEndpoint: SecuredEndpoint[Option[Instant], Unit, List[Notification], Any] =
+    baseNuApiEndpoint
+      .summary("Endpoint to display notifications")
+      .tag("Notifications")
+      .get
+      .in("notifications")
+      .in(query[Option[Instant]]("after").example(Some(Instant.parse("2023-07-29T19:30:00Z"))))
```

Comment:
it would be nice to add tests for `after` != None:
1. with date-time passed in UTC (like in the example)
2. with date-time passed in eg. our local date time
3. with date-time invalid format

Reply:
`after` parameter is removed altogether as it is not used anywhere